### PR TITLE
Updated README

### PR DESCRIPTION
**Pull request by CarloQuick**
Added sections and more links to the README.

Future steps - will work on linked docs once this PR is complete.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,9 +3,38 @@
...
+
+[Anki](https://apps.ankiweb.net) helps you learn faster with user created flashcards that makes learning easier and more efficient compared to traditional study methods.
+
+Card content is endless as they support:
+
+- images :framed_picture:
+- audio :loud_sound:
+- videos :film_strip:
+- scientific markup (Via LaTeX) :dna:
+
```
Comment (dae) [inline README.md:18]: Can we cut out the extra text here and just say something like "Anki is a spaced repetition program. Please see [the website](...) to learn more."? The emoji is a little more playful than the style we typically use, and "they support" sounds bad.

File: README.md [hunk 2]
```diff
@@ -3,9 +3,38 @@
 [![Build status](https://badge.buildkite.com/c9edf020a4aec976f9835e54751cc5409d843adbb66d043bd3.svg?branch=main)](https://buildkite.com/ankitects/anki-ci)
 
 This repo contains the source code for the computer version of
-[Anki](https://apps.ankiweb.net).
+[Anki](https://apps.ankiweb.net) for people who would like to help improve the code.
```
Comment (dae) [inline README.md:6]: I don't feel this is an improvement I'm afraid - people may want access to the code without wanting to improve it.

File: README.md [hunk 3]
```diff
@@ -3,9 +3,38 @@
...
+Want to contribute to Anki? Check out the [Contribution Guidelines](./docs/contributing.md).
+
+### Anki Contributors :clap:
+
+[CONTRIBUTORS](./CONTRIBUTORS)
+
+# License
 
-For more information on building, please see [Development](./docs/development.md).
+For Anki's licensing - [LICENSE](./LICENSE).
```
Comment (dae) [inline README.md:40]:
```suggestion
Anki's license: [LICENSE](./LICENSE).
```

File: README.md (hunk 1 above)
Comment (CarloQuick) [reply README.md:11]: Extra text removed and added recommended statement  describing Anki.

File: README.md (hunk 3 above)
Comment (CarloQuick) [reply README.md]: License section more concise per your recommendation.

**Comment by dae** [comment]
Thanks Carlo!

**Pull request merged**